pulsar-client/src/main/java/org/apache/pulsar/client/impl/schema/util/SchemaUtil.java

```diff
@@ -33,7 +33,16 @@
 
 public class SchemaUtil {
 
+    private static Boolean globalJsr310ConversionEnabled = null;
+
+    public static void setGlobalJsr310ConversionEnabled(Boolean globalJsr310ConversionEnabled) {
+        SchemaUtil.globalJsr310ConversionEnabled = globalJsr310ConversionEnabled;
+    }
+
     public static boolean getJsr310ConversionEnabledFromSchemaInfo(SchemaInfo schemaInfo) {
```

Comment:
Please rename this function to "getJsr310ConversionEnabled" as it additionally checks the global var.

I am ok with exposing `setGlobalJsr310ConversionEnabled` as this seems to be the only exception that clients need to override. If we have other such global configs to override, we need to consider adding "GlobalSchemaOverrideConfiguration" to maintain them in a better way.

Reply:
renamed